src/Symfony/Component/Notifier/Bridge/Bluesky/composer.json

```diff
@@ -23,6 +23,7 @@
         "php": ">=8.2",
         "psr/log": "^1|^2|^3",
         "symfony/http-client": "^6.4|^7.0",
+        "symfony/mime": "^7.1",
```

Comment:
6.4 and 7.0 should work too, not sure how we handle that @nicolas-grekas

Reply:
^6.4|^7.0 then

Reply:
but this should be move to require-dev, the dep is not mandatory